[Operator Versioning][Edge] Use call once to ensure thread safety

cccclai opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* __->__ #70217

Differential Revision: [D33246058](https://our.internmc.facebook.com/intern/diff/D33246058/)

Commit:
[Operator Versioning][Edge] Use call once to ensure thread safety



Differential Revision: [D33246058](https://our.internmc.facebook.com/intern/diff/D33246058/)

[ghstack-poisoned]

zhxchen17 commented on torch/csrc/jit/mobile/import.cpp:
```diff
@@ -226,7 +226,7 @@ class BytecodeDeserializer final {
   // upgrader will be applied at loading stage.
   uint64_t operator_version_;
-  std::atomic<bool> _upgrader_initialized{false};
+  std::once_flag _upgrader_initialized;
```
upgrader_initialized_ for naming consistency?

zhxchen17 reviewed (approved):
looks good. some non-blocking nit.

Commit:
Update on "[Operator Versioning][Edge] Use call once to ensure thread safety"


Differential Revision: [D33246058](https://our.internmc.facebook.com/intern/diff/D33246058/)

[ghstack-poisoned]
